evap/templates/base.html

```diff
@@ -239,7 +239,12 @@
                     placeholder: "{% trans "Please select..." %}",
                     minimumResultsForSearch: 15,});
             };
-            apply_select2($('select').not('.form-template select'));
+            apply_select2($('select').not('.form-template select').not('[disabled]'));
+
+            apply_select2_disabled = function(elem) {
+                elem.select2();
+            };
+            apply_select2_disabled($('select[disabled]').not('.form-template select'));
```

Comment:
i could, but i don't want to have the "Please select..." text when there is nothing the user can select ;)